apps/bixarena/app/project.json

```diff
@@ -46,15 +42,18 @@
         "command": "docker/bixarena/serve-detach.sh {projectName}"
       }
     },
+    "serve-model-api": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "uv run uvicorn src.model.models_api:app --host 0.0.0.0 --port 8000",
+        "cwd": "{projectRoot}"
+      }
+    },
```

Comment:
See remark about the anti-pattern. This approach is not good even for a short term solution as it increases the complexity of this project and reduces its maintainability.